### DISCUSSION TITLE
Problem with accessing wandb : permission denied

### DISCUSSION THREAD

**Discussion by Ukd1796** [alibaba/FederatedScope#590] [General]
When I set wandb use:True . I get access denied could someone help me with this ?
![wandb](https://user-images.githubusercontent.com/81182582/235622511-ef57e274-5af5-43e5-9ba9-f2631ac7c584.jpg)

**Comment by Ukd1796** [comment]
I was able to solve this after adding the wandb user_name and project name as well in my config.yaml file. As of now I am able to see all the system metrics realtime such as gpu usage, memory utilization. Can I see the clients train loss values, accuracies and other training parameters realtime as well using wandb as well and can someone help me with how can I do that ?

**Reply by rayrayraykk** [reply]
Hi, as you can see in https://github.com/alibaba/FederatedScope/blob/b3724109ae25363d2cd7116cc476043b162bedec/federatedscope/core/workers/client.py#L353 , please set `cfg.wandb.client_train_info` to `True`.
